content/docs/gto/install.md

```diff
@@ -0,0 +1,33 @@
+# Installation
+
+To create an Artifact Registry with GTO, you only need a Git repo and GTO
+package installed. There's no need to set up any services or databases, compared
+to many other Model Registry offerings.
+
+To check whether GTO is installed in your environment, run `which gto`. To check
+which version is installed, run `gto --version`.
+
+## Install as a Python library
+
+GTO is a Python library. It works on any OS. You can install it with a package
```

Comment:
We say that GTO is a command line tool in the cmd ref (closer to correct, I think). Here we say it's a library (misleading as its main use is via CLI).